relay-system/src/lib.rs

```diff
@@ -16,5 +16,7 @@
 
 pub mod compat;
 mod controller;
+mod service;
 
 pub use self::controller::*;
+pub use self::service::*;
```

Comment:
What would you say to make `pub mod service` and remove the `service::*` use?  This is how you use it in #1421 (and the missing `pub` is why that's currently unhappy) and it kind of makes sense (i'm generally not a fan of * imports)

Reply:
This was related to a comment by Jan:

> In this case, I'd rather make this a private module and do a blanket export like for controller. This module will contain the most used types of this crate, so it's worth having them in the root namespace.